### ISSUE TITLE
About the test set results

### ISSUE THREAD

**Issue by sgrrlll** [hailanyi/CasA#24]
Hello, I have made improvements to your baseline, and all three categories have improved on the validation set. However, the test results submitted to the kitti official website are much lower than your results. Could you please give me some suggestions?

**Comment by faziii0** [comment]
@sgrrlll have you sloved this problem

**Comment by hailanyi** [comment]
The score threshold used to submit to test set is different from validation set, as the KITTI metric is much more sensitive to object recall. Please use a lower score threshold as follows and train your detector on all training data to attain a good performance.
```
    POST_PROCESSING:
        RECALL_THRESH_LIST: [0.3, 0.5, 0.7]
        SCORE_THRESH: 0.3 #  using 0.1 if your results are still not good 
        OUTPUT_RAW_SCORE: False
```

**Comment by GP-Bone** [comment]
> The score threshold used to submit to test set is different from validation set, as the KITTI metric is much more sensitive to object recall. Please use a lower score threshold as follows and train your detector on all training data to attain a good performance.
> 
> ```
>     POST_PROCESSING:
>         RECALL_THRESH_LIST: [0.3, 0.5, 0.7]
>         SCORE_THRESH: 0.3 #  using 0.1 if your results are still not good 
>         OUTPUT_RAW_SCORE: False
> ```
Thanks for your suggestion. I have some doubts, when I use train+val to train the model, how do I pick the best model?
Or if I use 80% of the data and the checkpoint with the highest mAP doesn't work well in the test set, should I choose based on the maximum of the recall or judge from the PR curve. Looking forward to your reply.

![image](https://github.com/hailanyi/CasA/assets/111340137/cd6ed341-c38d-42b1-9b76-c49bf42d2dc6)
